flow/util/markDontUse.py

```diff
@@ -45,6 +45,12 @@
 content, count = re.subn(pattern, replace, content)
 print("Commented", count, "lines containing \"original_pin\"")
 
+# Yosys, does not like properties that start with : !, without quotes
```

Comment:
I know I'm necroposting, but I'm revisiting the necessity for this script from the yosys side. Is this a workaround for https://github.com/YosysHQ/yosys/issues/3498 which has been fixed?

cc @povik

Reply:
Highly likely, you can try removing those lines (file is currently called `preprocessLib.py`) and see if it still works as intended. I think I recall that it used to emit a warning that it was ignoring the cell, not error out.